src/main/java/org/spongepowered/mod/mixin/core/server/MixinServerConfigurationManager.java

```diff
@@ -89,6 +89,8 @@
 @Mixin(ServerConfigurationManager.class)
 public abstract class MixinServerConfigurationManager implements IMixinServerConfigurationManager {
 
+    private static final String $OVERWRITE_POLICY = "FORGE(1371 +10)";
```

Comment:
I like using $ to indicate that something is a structural member rather than something which should be read as part of the class